scripts/generators/beats.py

```diff
@@ -62,6 +62,14 @@ def fieldset_field_array(source_fields, df_whitelist, fieldset_prefix):
         if not ecs_field['flat_name'] in df_whitelist:
             beats_field['default_field'] = False
 
+        # Include the `path` attribute for `alias` types.
+        # https://github.com/elastic/ecs/issues/876
+        if ecs_field['type'] == 'alias':
```

Comment:
Instead of handling this here, let's add `path` to the `allowed_keys` list in line 34. When `beats_field = ecs_helpers.dict_copy_keys_ordered(ecs_field, allowed_keys)` is called in line 43, this should take care of the copying the `path` key/value over.

Reply:
Added in https://github.com/elastic/ecs/pull/877/commits/4fba68804468b321e5e12892235134d149ab75c8 :+1:

Given that `path` is (to my knowledge) only valid when `type: alias`, I wasn't sure if it was okay to include it in `allowed_keys`.